azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/domain/OSProfile.java

```diff
@@ -91,15 +90,58 @@ public static LinuxConfiguration create(final String disablePasswordAuthenticati
 
       @AutoValue
       public abstract static class WinRM {
+          public enum ListenerProtocol {
+
+              HTTP("http"),
+              HTTPS("https"),
+              UNRECOGNIZED("Unrecognized");
+
+              private String value;
+
+              ListenerProtocol(String value) {
+                 this.value = value;
+              }
+
+              public static ListenerProtocol fromValue(String value) {
+                 ListenerProtocol[] items = ListenerProtocol.values();
```

Comment:
Could use the same pattern as is in https://github.com/jclouds/jclouds-labs/blob/master/azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/domain/ComputeNode.java#L28